### PR TITLE
Add `coveralls` to `test-ci-prod` Github Action

### PR DESCRIPTION
**Pull request by 0xjei**
__Dependencies__
- [x] #2

__Resources__
- https://coveralls.io/
- https://coveralls.io/github/privacy-scaling-explorations
- https://github.com/marketplace/actions/coveralls-github-action

### CODE REVIEW

File: .github/workflows/test-ci-prod.yaml [hunk 1]
```diff
@@ -58,6 +58,15 @@ jobs:
         env:
           GOOGLE_APPLICATION_CREDENTIALS: ./packages/backend/serviceAccountKey.json
 
+      # run coveralls
```
Comment (gurrpi) [inline .github/workflows/test-ci-prod.yaml:61]: Modify `yarn test:ci-prod` command to enable coverage collecting

File: .github/workflows/test-ci-prod.yaml [hunk 2]
```diff
@@ -58,6 +58,15 @@ jobs:
...
+      - name: Coveralls
+        uses: coverallsapp/github-action@master
+        with:
+            github-token: ${{ secrets.GITHUB_TOKEN }}
+            flag-name: run-${{ matrix.type }}
```
Comment (gurrpi) [inline .github/workflows/test-ci-prod.yaml:66]: Remove this input. This workflow doesn't have `matrix.type` value

File: .github/workflows/test-ci-prod.yaml [hunk 3]
```diff
@@ -58,6 +58,15 @@ jobs:
...
+            path-to-lcov: ./coverage/${{ matrix.type }}/lcov.info
+            parallel: true
```
Comment (gurrpi) [inline .github/workflows/test-ci-prod.yaml:68]: Remove this input

**Review by gurrpi — CHANGES_REQUESTED** [review]
Can you please double-check the inputs for Coveralls action and ensure that they're appropriate for our project?

Thank you for your hard work and attention to detail. I appreciate your efforts in integrating this tool!

File: .github/workflows/test-ci-prod.yaml (hunk 2 above)
Comment (0xjei) [reply .github/workflows/test-ci-prod.yaml:66]: Ohh correct, I didn't see the `lcov.info` location under `coverage` without the need of `matrix.type`. Thanks for the heads up!

**Pull request merged**